### ISSUE TITLE
Model project peer feedback [vrx877]

### ISSUE THREAD

**Issue by vrx877** [NumEconCopenhagen/projects-2019-bcg#12]
Hi BCG!

A very comprehensive project, which runs neatly and smooth. Quite long, but a nice showcase of how to solve a model analytically through programming. 

Here's my feedback: 

**The best part of the project was..**
The code is and documentation is great. Very easy to follow. I am very impressed by the final graph, as well. Good that you threw this part in.

**The hardest part of the project to understand was..**
Going through the project is honestly a bit cumbersome. Not quite hard to understand, but it is a very long project, and a great part of it is just showcasing what SymPy can do. The expansion towards the end is good, but a bit confusing when unfamiliar with OLG-models. Consider expanding the description a little. 

I also found some of the functions you utilize a bit confusing and had to google e.g. Max_Iter.

**An idea for an improvement /clarification could be..**
Use a bit more comments and descriptions when your projects turn from simple to difficult. It's quite a big leap between the individual parts of the project and a bit incoherent.

**An idea for an extension could be..**
Maybe use numerical optimization to illustrate part of your points! I believe this is actually a requirement in the assignment, by using an optimizer. You could decrease the beginning and then calculate more implied results from the model, going forward.

In conclusion though, I am impressed with your notebook. Good work!
